packages/training/deploy/docker/scripts/entrypoint.sh

```
@@ -0,0 +1,90 @@
+        print(f'⚠ Database connection failed: {e}')
+
+asyncio.run(test())
+" || echo "⚠ Database connection test failed."
+    echo ""
+else
+    echo "⚠ DATABASE_URL not set - skipping database check"
+    echo ""
+fi
+
+# Run the command
+echo "Starting: $@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)